### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 15

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#341] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325203-ab6a753b-d7b4-47e0-9513-9e190830ed22.png)

**Comment by Exyss** [comment]
![Boh(22)](https://user-images.githubusercontent.com/65043207/152686777-ede514fa-7e75-4d0b-8ffe-b022d442df2e.jpg)
